templates/docs/patterns/buttons.md

```diff
@@ -123,6 +123,32 @@ In some contexts, it may be necessary to indicate to the user that a button is i
 View example of the contextual menu pattern
 </a></div>
 
+## Accessibility
+
+{# copy doc: https://docs.google.com/document/d/11H6xPtBRxB1a8KCmK7XFH1FeZDduQhSJXxq-wNJ_CZ8/edit#heading=h.qxwy2zql3yy7 #}
```

Comment:
Overall I'd prefer to have one source of truth, so once we merge it lets keep all the edits here. So I'm not sure if we need to link to the doc.

Reply:
@bethcollins92 Did you remove the link from the others?

Reply:
Ahh sorry I thought I'd done it from them all! Doing it now